cypress/integration/chi-js/chi-tabs.spec.js

```diff
@@ -0,0 +1,122 @@
+describe('Chi Tabs', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/tabs.html');
+  });
+//horizontal tab
+
+describe('horizontal tab contains sliding border', () => {
+it(' horizontal tab contains sliding border', () => {
+  
+cy.get('#example-horizontal-base li:nth-child(4)').should('have.class', 'chi-sliding-border');
+});
+});
+
+describe('Clicking horizontal tab 2 shows corresponding content 2 and active class is present',()=>{
+  it('Clicking horizontal tab 2 shows corresponding content 2 and active class is present', () => {
+    cy.get('#example-horizontal-base li:nth-child(2)')
+    .click()
+    .wait(550);
+   
+
+    cy.get('#horizontal-base-2').contains('Tab 2 content').should('be.visible');
+    
+    cy.get('#horizontal-base-2').should('have.class', '-active');
+    cy.get('#example-horizontal-base li:nth-child(4)')
+    .should('have.attr', 'style', 'left: 95.5938px; width: 56.625px;');
```

Comment:
```suggestion
    cy.get('#horizontal-base-2')
      .contains('Tab 2 content')
      .should('be.visible')
      .should('have.class', '-active');
```
Let's remove `.should('have.attr', 'style', 'left: 95.5938px; width: 56.625px;');` assertions  since changing the texts inside tabs will break these assertions if left and width properties are hardcoded.

Reply:
Removed the chi-border assertion

Reply:
removed the duplicate cy.get() method.